cluster/addons/fluentd-elasticsearch/fluentd-es-ds.yaml

```diff
@@ -64,9 +64,10 @@ spec:
       # This annotation ensures that fluentd does not get evicted if the node
       # supports critical pod annotation based priority scheme.
       # Note that this does not guarantee admission on the nodes (#40573).
```

Comment:
drop this comment, the related annotation no longer exists

Reply:
This still needs to be addressed

Reply:
Thanks for pointing it out, the comments are now removed.

Reply:
@liggitt @spiffxp PTAL